saltyrtc/client.ts

```diff
@@ -225,33 +197,42 @@ export class Client {
             }
 
             // Send offer
-            this.peerConnection.createOffer().then(
+            /*this.peerConnection.createOffer().then(
                 (offer) => {
                     this.peerConnection.setLocalDescription(offer); // TODO: Should we handle errors?
                     this._startConnectTimer();
                     this.signaling.sendOffer(offer);
                 },
                 (error) => console.error('PeerConnection error:', error)
-            );
-        });
-        this.$rootScope.$on('signaling:answer', (_, answer) => {
-            this.peerConnection.receiveAnswer(answer);
-        });
-        this.$rootScope.$on('signaling:candidate', (_, candidate) => {
-            this.peerConnection.receiveCandidate(candidate);
+            );*/
         });
+    }
 
-        // Listen for peer connection events and delegate them
-        this.$rootScope.$on('pc:candidate', (_, candidate) => {
-            this.signaling.sendCandidate(candidate);
-        });
+    /**
+     * Send an ICE candidate through the signalling channel.
+     *
+     * TODO: Make sure candidates are buffered for 10ms, according to the
+     * SaltyRTC spec.
+     */
+    public sendCandidate(candidate: RTCIceCandidate) {
```

Comment:
Interesting... the WebRTC spec has changed and is now using a JS dictionary. `RTCIceCandidate` may look different for ORTC. I hope this will not create any type errors.